src/style.css

```diff
@@ -1,3 +1,75 @@
+:root {
+  --link-color: #000;
+  --link-active-color: #00acdc;
+}
+
+html {
+  font-size: 10px;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 5rem;
+  align-items: center;
+}
+
+.logo__link {
+  display: block;
+  width: 4rem;
+  height: 4rem;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  font-size: 1.2rem;
+  padding-inline-start: 0;
+  margin: 0;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class`. Replace the `list-style` property from the `.nav__list` class to a class that specifically targets `li` elements.